grid-client/state/state.go

```diff
@@ -181,6 +180,16 @@ func (st *State) LoadK8sFromGrid(nodeIDs []uint32, deploymentName string) (workl
 		return workloads.K8sCluster{}, fmt.Errorf("failed to get master node for k8s cluster %s", deploymentName)
 	}
 	cluster.NodeDeploymentID = nodeDeploymentID
+	cluster.NetworkName = cluster.Master.NetworkName
+	cluster.SSHKey = cluster.Master.SSHKey
+	cluster.Token = cluster.Master.Token
+
+	// get cluster IP ranges
+	err := st.AssignNodesIPRange(&cluster)
```

Comment:
in order to load k8s we will need to have the cluster network already stored in state.
I think we can get the cluster network since we have its name and which nodes it is deployed on, that way we wouldn't depend on it being stored in state.

Reply:
good idea 🎆